changelog/unreleased/bugfix-ocis-load-only-supported-thumbnails

```diff
@@ -0,0 +1,6 @@
+Bugfix: OCIS Load only supported thumbnails
```

Comment:
```suggestion
Bugfix: Load only supported thumbnails (configurable)
```
Please don't mention oCIS in the changelog item. We will only use this in the default config of oCIS (needs to be done with a PR to oCIS repo), but it would be easily usable with oc10 backend as well.